### DISCUSSION TITLE
lock statements without explicit curly braces

### DISCUSSION THREAD

**Discussion by DevAM-Tools** [dotnet/csharplang#8252] [Language Ideas]
I would like propose `lock` statements without explicit curly braces similar to `using` statements without explicit curly braces. This might help to reduce the amount of indentation when working with locks.

Example:
```Csharp
void Foo()
{
  // Do some other stuff
  lock(_lockObject)
  {
    // Do some stuff
  }
}

```
would become
```CSharp
void Foo()
{
  // Do some other stuff
  lock(_lockObject);
  // Do some stuff
}
```
while the lock gets release at the end of the current block.

**Comment by FredyFerrari** [comment]
IMO, lock (and also try except statements) are critical elements and I'd prefer to keep it as clear as possible, where the lock starts and ends (and what's inside of a try except statement).

**Comment by 333fred** [comment]
Duplicate of https://github.com/dotnet/csharplang/discussions/2451.
